doc/rfcs/0002-support-run-component-without-tiup.md

```diff
@@ -0,0 +1,23 @@
+# Support Run Component without TiUP
+
+## Summary
+
+Allow users to only use TiUP during installation and upgrade, without needing to invoke TiUP while using the component.
```

Comment:
Maybe we can set a version per component like this:
```
tiup version set ctl v7.2.0
```

And then have `tiup ctl ...` always use v7.2.0. The default should be `auto`/`latest`. And then have `tiup version show` to list all components and their set vesion.

Reply:
I don't want to increase the complexity of using tiup or replace the old way of using it. Users could still use it like "tiup cluster:v1.11.0" to choose versions. It is a simple way to use soft link rather than maintenance selected version